packages/ui-tests/stories/settings/AboutModal.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { KaotoAboutModal } from '@kaoto/kaoto/testing';
+import { Meta, StoryFn } from '@storybook/react';
+import { useState } from 'react';
+
+export default {
+  title: 'Settings/KaotoAboutModal',
+  component: KaotoAboutModal,
+} as Meta<typeof KaotoAboutModal>;
+
+const Template: StoryFn<typeof KaotoAboutModal> = (args) => {
+  const [isModalOpen, setIsModalOpen] = useState(true);
+  const handleClose = () => setIsModalOpen(!isModalOpen);
+  return (
+    <>
+      <button onClick={() => setIsModalOpen(true)}>Open Kaoto About Modal</button>
+      <KaotoAboutModal {...args} handleCloseModal={handleClose} isModalOpen={isModalOpen} />
```

Comment:
mmm, for some reason the modal is not open :thinking:

Reply:
yes, that's strange, for some reason, the snapshot taken displayed the button, instead of the modal. Removing the button seems to fix the problem